From c99f6759f1a1e965545525aea748e7e2349e8f2f Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Tue, 18 Feb 2020 14:17:26 +0200
Subject: [PATCH 6/6] Revert "delete AudioFX"

This reverts commit dc95b1bc29b2f4309bfd90693ce6a3b427af107c.

	modified:   config/common.mk
---
 config/common.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/common.mk b/config/common.mk
index 10a68ff4..3461d9b3 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -130,6 +130,7 @@ PRODUCT_PACKAGES += \
 
 # Lineage packages
 PRODUCT_PACKAGES += \
+    AudioFX \
     Backgrounds \
     LineageParts \
     LineageSettingsProvider \
-- 
2.17.1

